view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -193,8 +274,13 @@ define(
                         'expiryMonth': this.expiryMonth(),
                         'expiryYear': this.expiryYear(),
                         'holderName': this.creditCardOwner(),
-                        'store_cc': this.setStoreCc(),
-                        'number_of_installments': this.installment()
+                        'store_cc': this.setStoreCc,
+                        'number_of_installments': this.installment(),
+                        'java_enabled': navigator.javaEnabled().toString(),
```

Comment:
please use already available 3DS2 helper functions. It will be easy and consistent across all plugins.
https://github.com/Adyen/adyen-3ds2-js-utils

Reply:
Thank you for the heads up, I didn't know that there is this helper but definitely it would have been easier to use it :) I will just change the current functions to this.